Builds/containers/gitlab-ci/pkgbuild.yml

```diff
@@ -131,6 +132,65 @@ dpkg_build:
   script:
     - . ./Builds/containers/gitlab-ci/build_package.sh dpkg
 
+#########################################################################
+##                                                                     ##
+##  stage: sign_packages                                               ##
+##                                                                     ##
+##  build packages using containers from previous stage.               ##
+##                                                                     ##
+#########################################################################
+
+rpm_sign:
+  stage: sign_packages
+  dependencies:
+    - build_centos_container
```

Comment:
if we want to sign the packages that were built in the previous step, seems like these should depend on `rpm_build/dpkg_build` instead.